src/main/java/net/dries007/tfc/objects/te/TECrucible.java

```diff
@@ -68,7 +66,7 @@ public TECrucible()
     {
         super(10);
 
-        this.alloy = new Alloy(CRUCIBLE_MAX_METAL_FLUID);
+        this.alloy = new Alloy(ConfigTFC.GENERAL.tankCrucible); // Side effect: Maximum amount only matches config if not loading from disk
```

Comment:
See above comment re: data fixer on config option changes

Reply:
Made Crucibles void the surplus instead to match barrels.